packages/venia-concept/src/ServiceWorker/registerRoutes.js

```diff
@@ -0,0 +1,53 @@
+import {
+    isResizedCatalogImage,
+    findSameOrLargerImage
+} from './Utilities/imageCacheHandler';
+import { createCatalogCacheHandler } from './Utilities/catalogCacheHandler';
+import { thirtyDays } from './defaults';
+
+export default function() {
```

Comment:
I had to make this a function to accommodate the fact that the workbox object was loading after the JS Engine has parsed through these lines of code.